src/app/components/input-amount/input-amount.component.ts

```diff
@@ -4,14 +4,14 @@ import {
 	FormControl,
 	FormGroupDirective,
 } from "@angular/forms";
+import { BigNumber } from "@arkecosystem/platform-sdk-support";
```

Comment:
This doesn't seem to be a dependency in the `package.json` file.

Reply:
Pushed it just now on the last commit.